docs/src/pages/components/autocomplete/UseAutocomplete.tsx

```diff
@@ -1,6 +1,6 @@
 /* eslint-disable @typescript-eslint/no-use-before-define */
 import * as React from 'react';
-import { useAutocomplete } from '@material-ui/unstyled/AutocompleteUnstyled';
+import useAutocomplete from '@material-ui/core/useAutocomplete';
```

Comment:
If we use https://next.material-ui.com/components/switches/#unstyled-switches as a baseline template for the docs, shouldn't we use unstyled here? 
```suggestion
import { useAutocomplete } from '@material-ui/unstyled/AutocompleteUnstyled';
```

Reply:
But yeah, it could always be done in a follow-up to restructure the docs https://next.material-ui.com/components/autocomplete/#useautocomplete, emphasizing on unstyled.

Reply:
Yes, absolutely. I just missed this.